src/status_im/ui/screens/wallet/navigation.cljs

```diff
@@ -1,12 +1,20 @@
 (ns status-im.ui.screens.wallet.navigation
   (:require [status-im.constants :as constants]
-            [status-im.ui.screens.navigation :as navigation]))
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.ui.screens.wallet.signing-phrase.views :as signing-phrase]))
 
 (def transaction-send-default
   {:method constants/web3-send-transaction
    :symbol :ETH})
 
-(defmethod navigation/preload-data! :wallet-request-transaction
+(defmethod navigation/preload-data! :wallet-stack
```

Comment:
Still use that? No params passing?

Reply:
yes, we want to show "signing phrase" when open wallet tab each time